testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hudi/hive.properties

```diff
@@ -0,0 +1,14 @@
+connector.name=hive
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.allow-drop-table=true
```

Comment:
Just FYI this file would be only relevant in the context of doing hive<-> hudi  redirections.
It is quite likely that this file can be omitted at the moment.

Reply:
We should use `hive.security=allow-all`